lib/helpers/render_all_sub_topics.rb

```diff
@@ -1,9 +1,14 @@
 require 'pathname'
 
 module RenderAllSubTopics
-  def render_all_sub_topics(topic_name)
+  def render_all_sub_topics(topic_name, options={})
     output = ActiveSupport::SafeBuffer.new
-    Dir["#{@app.source_dir}/includes/#{topic_name}/_*"].sort.collect do |file|
+    files = Dir["#{@app.source_dir}/includes/#{topic_name}/_*"].sort
+    if options[:reverse_order]
```

Comment:
This was needed because the changelog files need to be rendered in the reverse direction (latest release first).